classes/core/PKPRequest.inc.php

```diff
@@ -591,6 +591,18 @@ function &getUser() {
 	function getUserVar($key) {
 		$_this = PKPRequest::_checkThis();
 
+		if (is_a($this, 'Request')) {
```

Comment:
@kaschioudi, are you sure? I don't think that's possible, looking at the source of `PKPRequest::_checkThis`, unless the `APIProfileForm` somehow gets stored as the request in the registry (which I doubt is happening).

Off the top of my head, I wonder if you aren't mixing `$_this` and `$this` in the function by accident. Using `$_this` (as returned from `_checkThis`) should be OK.